config/config.go

```diff
@@ -81,7 +81,7 @@ func Load(rootDir string) (*Config, error) {
 
 	err = yaml.Unmarshal(yamlContent, &config.overrides)
 	if err != nil {
-		return nil, err
```

Comment:
You are dropping the original yaml error here. Instead, wrap it so the details aren't lost: https://stackoverflow.com/a/61284074

Same for the other changes

Reply:
check it out

Reply:
as per the link above, I think you need to use `%w` rather than `%v`

Reply:
ok